### PR TITLE
Fix #541

### LINKED ISSUE #541: VARIABLES.md - secrets_protection_enabled description
https://github.com/confluentinc/cp-ansible/blob/6.0.1-post/VARIABLES.md#secrets_protection_enabled
Must be except **Zookeeper** and Rest Proxy

### PR DESCRIPTION
**Pull request by hifly81**
# Description

Fix https://github.com/confluentinc/cp-ansible/issues/541

Fixes # (issue)
https://github.com/confluentinc/cp-ansible/issues/541

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

**Comment by JumaX** [comment]
@hifly81 Good catch!  Can you point this to the 6.0.x branch.  We generally don't update -post branches, they are meant to be static releases.

**Comment by JumaX** [comment]
@hifly81 Apologies, should have caught this earlier, but you actually need to make this change in the variables role, defaults, main.yml line 948.  Then you can run python doc.py to update the variables.md and commit.

**Comment by hifly81** [comment]
> @hifly81 Apologies, should have caught this earlier, but you actually need to make this change in the variables role, defaults, main.yml line 948. Then you can run python doc.py to update the variables.md and commit.

File: VARIABLES.md [hunk 1]
```diff
@@ -1510,7 +1510,7 @@ Default:  "{{rbac_component_additional_system_admins}}"
 
 ### secrets_protection_enabled
 
-Boolean to enable secrets protection on all components except Rest Proxy
+Boolean to enable secrets protection on all components except Zookeeper and Rest Proxy
```
Comment (domenicbove) [inline VARIABLES.md:1513]: Can you also remove RP from the exception? it works on RP

**Comment by hifly81** [comment]
> @hifly81 Good catch! Can you point this to the 6.0.x branch. We generally don't update -post branches, they are meant to be static releases.

done

**Review by JumaX — APPROVED** [review]
LGTM.

**Pull request merged**